### PR TITLE
proposals/distribution: remove non-sense about image indexes

### PR DESCRIPTION
**Pull request by stevvooe**
Signed-off-by: Stephen J Day <stephen.day@docker.com>

### CODE REVIEW

**Comment by dmcgowan** [comment]
LGTM

File: proposals/distribution.md [hunk 1]
```diff
@@ -112,10 +112,8 @@ The following entries should be added to the [scope table][scope]:
     * What: Specifying a distribution method
     * In/Out/Future: In scope
     * Status: In progress (see opencontainers/distribution-spec)
-    * Description: Define a protocol for creating, retrieving, updating, and deleting objects defined in the [image specification][image-spec].
-        Listing repositories (like [`/v2/_catalog`][catalog]) is a multi-[image-index][] action, which is out of scope for this entry.
-        Managing groups of image indexes requires multi-[image-index][] actions, which are out of scope for this entry.
-        Listing image indexes within a group is a multi-[image-index][] action, which is out of scope for this entry.
+    * Description: Define a protocol for creating, retrieving, updating, and deleting content-addressable objects, such as those defined in the [image specification][image-spec].
+		Listing repositories is a registry-wide operation; the implementation of which is out of scope of this document.
```
Comment (mikebrow) [inline proposals/distribution.md]: Nit: this is a first introduction of term "document." Suggest using this "spec."
Also this listing restriction does not make sense here. Can we move it to the Grouping line above on 109 and change the word grouping to a broader term like "managing?"

Comment (dmcgowan) [reply proposals/distribution.md]: The relationship between these two sentences is not straightforward. However I do think this is referring to slightly different functionality. While the "grouping" section is just saying that each repository name is considered independent from the perspective of the API, this is referring to an endpoint which lists all the repository names within a registry. While the group and name hierarchy could be of interest to an end API user, cataloging has normally just be considered an administrative endpoint not exposed to API users.

+1 on document -> specification

Comment (mikebrow) [reply proposals/distribution.md]: Cool..
Maybe.. "An endpoint for listing repository names within a registry, is considered out of scope for this spec and thus also out of scope for this protocol."

Comment (dmcgowan) [reply proposals/distribution.md]: That language sounds clearer, let me update and move it next to tag listing. I don't think these statements belong here.

**Comment by dmcgowan** [comment]
Pulled this commit into #50, please close this PR

**Comment by crosbymichael** [comment]
closing per request with changes in #50

**Pull request closed without merging**